examples/gnrc_border_router/README.md

```diff
@@ -1,4 +1,144 @@
+This tool is found in `/dist/tools/uhcp`. So, as for `ethos`:
+
+```bash
+make clean all
+```
+
+Then you can compile and flash your `gnrc_border_router` example.
+To select ethos as the serial driver, be sure that the `Makefile` has the following:
+
+```make
+ifeq (,$(filter native,$(BOARD)))
+GNRC_NETIF_NUMOF := 2
+USEMODULE += ethos gnrc_netdev2
+CFLAGS += '-DETHOS_UART=UART_DEV(0)' -DETHOS_BAUDRATE=115200 -DUSE_ETHOS_FOR_STDIO
+FEATURES_REQUIRED += periph_uart
+endif
+# include UHCP client
+USEMODULE += gnrc_uhcpc
+```
+
+Then, proceed to compile and flash `gnrc_border_router` to your board as usual:
+
+```bash
+make clean all flash
+```
+
+## Usage
+Start the `start_network.sh` script by doing on `dist/tools/ethos`:
+
+```bash
+sudo sh start_network.sh /dev/ttyACMx tap0 2001:db8::/64
+```
+
+This will execute the needed commands to setup the network interface and configure the BR.
+Notice that this will also configure `2001:db8::/64` as a prefix.
+This prefix should be announced to other motes through the wireless interface.
+
+At this point you should be able to ping motes using the global address.
+You can do this through the RIOT console, for instance using `gnrc_networking`:
+
+```
+> ping6 2001:db8:0:1234:0:567:8:1
+```
+
+Just replace this address by your mote's address.
+Using `ifconfig` shows you the addresses of your mote.
+
+The script also sets up a link-local address on your Linux network interface.
+It can be pinged from 6lo motes:
+
+```
+> ping6 fd00:dead:beef::1
+```
+
+Thus far, you can communicate using 6lowPAN and IPv6 between motes and your PC.
+
+## Issues
+A common issue is that after ~15min global addresses on the BR Neighbour Cache dissappear.
+We are investigating why it happens and we'll propose a solution soon.
+If you experience it, just reboot your nodes and they will come again on-line.
+
+# gnrc_border_router with manual config
+You can use `ethos` as a standalone driver, if you want to setup the BR manually.
+
+## Setup
+As for the automatic setup, be sure that your `Makefile` is using the `ethos` driver.
+Then compile it as explained previously.
+
+Afterwards, you need IPv6 access to the nodes attached to the BR from your linux machine.
+To do this, it is necessary to add a `tap` interface.
+As an example, you can do it as follows:
+
+```bash
+sudo ip tuntap add tap0 mode tap user ${USER}
+```
+
+This will setup your `tap` interface.
+Then configure it to route the packets coming from your 6lo network:
+
+```bash
+sudo ip link set tap0 up
+sudo ip a a 2001:db8::1/48 dev tap0
+sudo ip r d 2001:db8::/48 dev tap0
+sudo ip r a 2001:db8::2 dev tap0
+sudo ip r a 2001:db8::/48 via 2001:db8::2 dev tap0
+```
+
+Then you can flash the `gnrc_border_router` example on your board:
+
+```bash
+make clean all flash
+```
+
+On this RIOT BR, add some addresses and a route:
+
+```
+> ifconfig 6 add 2001:db8::2/48
+> ifconfig 5 add 2001:db8::3/64
+> fibroute add :: via <link-local of tap> dev 6
```

Comment:
You should tell what type of interface are 5 and 6 (wired/wireless). Looking at haw the BR is configured by `start_network.sh`, the BR wired interface has a link-local address `fe80:`